tests/longevity-in-memory-36GB-4days.yaml

```diff
@@ -0,0 +1,81 @@
+test_duration: 5760
+prepare_write_cmd: "cassandra-stress write cl=QUORUM n=36000000 -schema 'replication(factor=3)' -port jmx=6868 -mode cql3 native -rate threads=500 -pop seq=1..36000000"
+stress_cmd: ["cassandra-stress mixed cl=QUORUM duration=5760m -schema 'replication(factor=3)' -port jmx=6868 -mode cql3 native -rate threads=20 -pop seq=1..36000000 -log interval=5",
```

Comment:
This translated to much less than 36GB.
You don't specify the column size, so it takes the default.
either increase the number of keys or specify "-col -size"